code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -104,7 +104,10 @@
 
 	if(gib_on_success)
 		new_xeno.visible_message("<span class='danger'>[new_xeno] bursts out of [owner] in a shower of gore!</span>", "<span class='userdanger'>You exit [owner], your previous host.</span>", "<span class='hear'>You hear organic matter ripping and tearing!</span>")
-		owner.gib(TRUE)
+		for(var/obj/item/bodypart/BP in owner.bodyparts)
+			if(BP.name == "chest")
+				BP.dismember()
+				break
 	else
 		new_xeno.visible_message("<span class='danger'>[new_xeno] wriggles out of [owner]!</span>", "<span class='userdanger'>You exit [owner], your previous host.</span>")
 		owner.adjustBruteLoss(40)
```

Comment:
Why not just change the gib_on_success thing? It's what we've done previously and it worked good. Dismemberment with the lack of cloning makes you as good as dead

Reply:
I also had a decent RP thanks to just not outright dying after the alien pops out, so that helps too